photon-client/src/components/cameras/CameraCalibrationInfoCard.vue

```diff
@@ -97,197 +95,221 @@ const importCalibration = async () => {
-          ><span class="text-no-wrap"
-            >{{ useCameraSettingsStore().currentCameraName }}@{{ getResolutionString(videoFormat.resolution) }}</span
-          ></v-card-title
-        >
-      </v-col>
-      <v-col>
-        <v-btn color="secondary" class="mt-4" style="width: 100%" @click="openUploadPhotonCalibJsonPrompt">
-          <v-icon left> mdi-import</v-icon>
-          <span>Import</span>
-        </v-btn>
-        <input
-          ref="importCalibrationFromPhotonJson"
-          type="file"
-          accept=".json"
-          style="display: none"
-          @change="importCalibration"
-        />
-      </v-col>
-      <v-col>
-        <v-btn
-          color="secondary"
-          class="mt-4"
-          :disabled="useCameraSettingsStore().getCalibrationCoeffs(props.videoFormat.resolution) === undefined"
+  <div>
+    <v-card color="primary" class="pa-6" dark>
+      <v-row>
+        <v-col cols="12" md="5">
+          <v-card-title class="pl-0 ml-0"
+            ><span class="text-no-wrap" style="white-space: pre !important">Calibration Details: </span
+            ><span class="text-no-wrap"
+              >{{ useCameraSettingsStore().currentCameraName }}@{{ getResolutionString(videoFormat.resolution) }}</span
+            ></v-card-title
+          >
+        </v-col>
+        <v-col>
+          <v-btn color="secondary" class="mt-4" style="width: 100%" @click="openUploadPhotonCalibJsonPrompt">
+            <v-icon left> mdi-import</v-icon>
+            <span>Import</span>
+          </v-btn>
+          <input
+            ref="importCalibrationFromPhotonJson"
+            type="file"
+            accept=".json"
+            style="display: none"
+            @change="importCalibration"
+          />
+        </v-col>
+        <v-col>
+          <v-btn
+            color="secondary"
+            class="mt-4"
+            :disabled="useCameraSettingsStore().getCalibrationCoeffs(props.videoFormat.resolution) === undefined"
+            style="width: 100%"
+            @click="downloadCalibration"
+          >
+            <v-icon left>mdi-export</v-icon>
+            <span>Export</span>
+          </v-btn>
+        </v-col>
+      </v-row>
+      <v-row
+        v-if="useCameraSettingsStore().getCalibrationCoeffs(props.videoFormat.resolution) !== undefined"
+        class="pt-2"
+      >
+        <v-card-subtitle>Calibration Details</v-card-subtitle>
+        <v-simple-table dense style="width: 100%" class="pl-2 pr-2">
+          <template #default>
+            <thead>
+              <tr>
+                <th class="text-left">Name</th>
+                <th class="text-left">Value</th>
+              </tr>
+            </thead>
+            <tbody>
+              <tr>
+                <td>Fx</td>
+                <td>
+                  {{
+                    useCameraSettingsStore()
+                      .getCalibrationCoeffs(props.videoFormat.resolution)
```

Comment:
`useCameraSettingsStore().getCalibrationCoeffs(props.videoFormat.resolution)` is used a lot here and can be made a computed property instead of calling this over and over every time. Also, more readable.